themes/bootstrap3/templates/Recommend/SideFacets/single-facet.phtml

```diff
@@ -33,30 +33,48 @@
 <?php else: ?>
-  <a href="<?=$toggleUrl ?>" class="<?=implode(' ', $classList) ?>" data-title="<?=$this->escapeHtmlAttr($this->facet['displayText']) ?>" data-count="<?=$this->facet['count'] ?>"<?php if($this->facet['isApplied']): ?> title="<?=$this->transEscAttr('applied_filter') ?>"<?php endif;?> data-lightbox-ignore>
+  <a href="<?=$toggleUrl ?>"
+     class="<?=implode(' ', $classList) ?>"
+     data-title="<?=$this->escapeHtmlAttr($this->facet['displayText']) ?>"
+     data-count="<?=$this->facet['count'] ?>"
+     title="<?php if($this->facet['isApplied']): ?><?=$this->transEsc('applied_filter') ?> - <?=$this->transEsc('page_reload_on_deselect_hint', ['%%filter_name%%' => $this->facet['displayText']])?><?php else: ?><?=$this->transEsc('page_reload_on_select_hint', ['%%filter_name%%' => $this->facet['displayText']])?>
+     <?php endif;?>"
+     data-lightbox-ignore>
 <?php endif; ?>
 
   <?php if ($hasSubLinks): ?>
-    <a class="text" href="<?=$toggleUrl ?>" data-lightbox-ignore data-title="<?=$this->escapeHtmlAttr($this->facet['displayText']) ?>" data-count="<?=$this->facet['count'] ?>"<?php if($this->facet['isApplied']): ?> title="<?=$this->transEscAttr('applied_filter') ?>"<?php endif;?>>
+    <a class="text" href="<?=$toggleUrl ?>" data-lightbox-ignore data-title="<?=$this->escapeHtmlAttr($this->facet['displayText']) ?>" data-count="<?=$this->facet['count'] ?>"
+        title="<?php if($this->facet['isApplied']): ?><?=$this->transEsc('applied_filter') ?> - <?=$this->transEsc('page_reload_on_deselect_hint', ['%%filter_name%%' => $this->facet['displayText']])?> <?=$this->escapeHtmlAttr($this->facet['displayText']) ?><?php else: ?><?=$this->transEsc('page_reload_on_select_hint', ['%%filter_name%%' => $this->facet['displayText']])?><?php endif;?>">
       <?=$displayText ?>
+      <span class="sr-only">
+        <?php if($this->facet['isApplied']): ?>(<?=$this->transEsc('applied_filter')?> - <?=$this->transEsc('page_reload_on_deselect_hint', ['%%filter_name%%' => $this->facet['displayText']])?> <?=$this->escapeHtmlAttr($this->facet['displayText']) ?>)<?php else: ?>(<?=$this->transEsc('page_reload_on_select_hint', ['%%filter_name%%' => $this->facet['displayText']])?>)<?php endif;?>
+      </span>
     </a>
   <?php else: ?>
     <span class="text">
       <?=$displayText ?>
+      <span class="sr-only">
+        <?php if($this->facet['isApplied']): ?>
+          (<?=$this->transEsc('applied_filter')?> - <?=$this->transEsc('page_reload_on_deselect_hint', ['%%filter_name%%' => $this->facet['displayText']])?>)<?php else: ?>(<?=$this->transEsc('page_reload_on_select_hint', ['%%filter_name%%' => $this->facet['displayText']])?>)
+        <?php endif;?>
+      </span>
     </span>
   <?php endif; ?>
 
   <?php if (!$this->facet['isApplied']): ?>
     <span class="badge">
-      <?=$this->localizedNumber($this->facet['count']) ?>
+      <?=$this->localizedNumber($this->facet['count']) ?><span class="sr-only"><?=$this->transEsc('results') ?></span>
```

Comment:
This seems like a problem for two reasons: 1.) I don't think there will be a space between the number and the word "results" and 2.) it would be better to use a string containing a token so that the order will be correct in languages where "results" might come before the number (which I believe is possible).

Reply:
1. The span element should cause a screen reader to read this as two words however.
2. A tokenized string would be better for translation. Perhaps as a title or aria-label on the `<span class="badge">` parent.